internal/pkg/agent/install/svc.go

```diff
@@ -57,25 +58,30 @@ func newService(topPath string, opt ...serviceOpt) (service.Service, error) {
 		o(&opts)
 	}
 
+	option := map[string]interface{}{
+		// GroupName
+		"GroupName": opts.Group,
+
+		// Linux (systemd) always restart on failure
+		"Restart": "always",
+
+		// Windows setup restart on failure
+		"OnFailure":              "restart",
+		"OnFailureDelayDuration": "15s", // Matches the value used by endpoint-security.
+		"OnFailureResetPeriod":   10,
+	}
+	if opts.Password != "" {
+		option["Password"] = opts.Password
```

Comment:
I'm pretty sure this password is stored encrypted in the Registry.  We need to document how the user would go about changing it.  It's not uncommon for password policies to require changing static passwords like this on a regular basis.

Reply:
I don't know if that is true in this case. The user cannot login at all, it can only be used as a service and the password is setup to never expire.

Reply:
> I don't know if that is true in this case.

See things like: https://www.stigviewer.com/stig/windows_server_2019/2020-06-15/finding/V-93209

Reply:
That is interesting finding, but even in this case the installing administrator doesn't even know the password. Being the finding is speaking about an administrator knowing the password, and in this case the password is completely unknown.

We can definitely add documentation about it for unprivileged installation on Windows. If they choose to change the user account password they will also have to update the password for the service.